data/tribes/buildings/productionsites/atlanteans/fishers_house/init.lua

```diff
@@ -25,7 +25,6 @@ tribes:new_productionsite_type {
    },
 
    aihints = {
-      collects_ware_from_map = "fish",
```

Comment:
I am lost now, how will fishers and other special buildings identified without this hint?

Reply:
Ou, I got it, this indeed looks like very good change...